src/main/java/org/team1540/robot2023/commands/arm/SetExtension.java

```diff
@@ -0,0 +1,40 @@
+package org.team1540.robot2023.commands.arm;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import org.team1540.robot2023.Constants;
+import org.team1540.robot2023.utils.RollingAverage;
+
+public class SetExtension extends CommandBase {
+    private final double threshold = 10;
+    private final RollingAverage rollingAverage = new RollingAverage(10);
+
+    private final Arm arm;
+
+    private final double extension;
+
+    public SetExtension(Arm arm, double x, double y) {
```

Comment:
it seems kind of counterintuitive to set the extension of an arm based on x-y coords and not actually travel to those coords, I think that it would make more sense to just set to a specific amount of extension length